tests/utils.go

```diff
@@ -2231,6 +2231,10 @@ func NewInt32(x int32) *int32 {
 	return &x
 }
 
+func NewInt64(x int64) *int64 {
```

Comment:
k8s has a nice library for all these pointers, `k8s.io/utils/pointer`

Reply:
we should replace NewBool and NewInt32 in this module then, ack